services/claim-backend/config/reward.ts

```diff
@@ -3,4 +3,5 @@ import { registerAs } from '@nestjs/config';
 export default registerAs('reward', () => ({
   communitySiteApiUrl: process.env.COMMUNITY_SITE_API_URL,
   delegationApiUrl: process.env.DELEGATION_API_URL,
+  graphQLClaimUrl: `${process.env.GRAPH_QL_CLAIM_ENDPOINT}/subgraphs/id/${process.env.CLAIM_SUBGRAPH_ID}`,
```

Comment:
We already have a config value for this URL in `indexer.ts`

Reply:
Yeah that's for the staking contract, this one is for the claim, but I'm moving this there.

Reply:
I mean let's also add this claim subgraph url in the indexer file.

Reply:
And it can include the subgraph id too

Reply:
Oh. Yeah. Nevermind